README.md

```diff
@@ -57,6 +57,36 @@ Warning: LFS client uses HTTP basic authentication, so using **HTTPS** is a must
 ./lfs_server.sh -verbose -pam login -s IP_ADDRESS -p PORT -cert domain.crt -key domain.key
 ```
 
+## Example
+
+```
+# Start a LFS server
+./lfs_server.sh
+
+# Clone a repo
+git clone ....
+cd repo
+
+# Add a normal file
+touch test.txt
+git add test.txt
+git commit -m "normal file"
+
+# Add a lfs file 
+git lfs install
```

Comment:
git lfs command is not available by default, you have to install git lfs client first. Maybe need to add a note about it.